### DISCUSSION TITLE
Weird solution for shutting down all chrome processes

### DISCUSSION THREAD

**Discussion by KonradOlech** [ultrafunkamsterdam/undetected-chromedriver#1731] [General]
Hi,
I was working on fixing issue with uc leaving Chrome processes open after closing script.
By accident I discovered that if you write function which calls driver.close() and DON'T INVOKE IT ANYWHERE then uc will close all Chrome processes correctly.

This is my function:

` 
def close_driver(driver):
driver.close()
`

When I try to call it, it throws an urllib error, when I try to call close() on driver directly then it leaves processes.

Weird fix, but working for me. Hopefully it will work with your code too.

**Comment by terra-alex** [comment]
Is this a way to avoid memory leaks like the ones mentioned in #1821 ? This sounds like a great solution if so

**Reply by Connor9994** [reply]
Looking for the exact same thing.
